src/server/pegasus_server_impl_init.cpp

```diff
@@ -86,6 +86,129 @@ METRIC_DEFINE_counter(replica,
                       dsn::metric_unit::kRequests,
                       "The number of abnormal read requests for each replica");
 
+METRIC_DEFINE_counter(replica,
+                      throttling_rejected_read_requests,
+                      dsn::metric_unit::kRequests,
+                      "The number of rejected read requests by throttling for each replica");
```

Comment:
If it's already define as `replica` level, is it some kind of duplicate to add "for each replica"?